server/test/services/program/ProgramServiceTest.java

```diff
@@ -459,6 +459,22 @@ public void validateProgramDataForCreate_requiresSlug(String adminName) {
                 "A program URL may only contain lowercase letters, numbers, and dashes"));
   }
 
+  @Test
+  @Parameters({"123", "235", "56789"})
+  public void validateProgramDataForCreate_requiresSlugBeingAlphanumerical(String adminName) {
+    ImmutableSet<CiviFormError> result =
+        ps.validateProgramDataForCreate(
+            adminName,
+            "display name",
+            "display desc",
+            "https://usa.gov",
+            DisplayMode.PUBLIC.getValue(),
+            ImmutableList.copyOf(new ArrayList<>()));
+
+    assertThat(result)
+        .containsExactly(CiviFormError.of("A program URL must contain at least one letter"));
+  }
+
```

Comment:
Can we also add a test that tests that it passes with one letter?

We can append something like `_failsIfNoLetters` and `_validIfOneLetter` to the end of the test names to disambiguate.

Reply:
for sure, a test with it passes with one letter added!